spec/helpers/static_pages_helper_spec.rb

```diff
@@ -1,15 +0,0 @@
-require 'rails_helper'
```

Comment:
Because we are not testing helpers. If we need to test the helper at some point, we can add the file back.